Nodes/Terminal.c

```diff
@@ -99,7 +99,7 @@ int GetChar(void) {
 }
 
 int handle1bChar(void) {
-  unsigned char ch;
+  char ch;
```

Comment:
It's was not needed as there is no comparison with constants > 127 there, but I'll update them anyway for consistency.